classes-recipe/tegra-uefi-capsule-signing.bbclass

```diff
@@ -16,9 +16,11 @@ UEFI_CAPSULE_TRUSTED_PUBLIC_CERT ?= "${PYTHON_BASETOOLS}/Pkcs7Sign/TestRoot.pub.
 
 # Override this function to have a secure signing server
 # perform the capsule signing.
-sign_uefi_capsules() {
+generate_uefi_capsule() {
```

Comment:
Structurally I like what has been done, but it is going to introduce breaking changes for anyone who has overridden `sign_uefi_capsules`.  The effect of renaming the class function is to not invoke someone's overridden signed implementation leaving capsules signed with default keys.  Capsule updates will fail.

Reply:
There are several things that make this a breaking change:  the class function name change, class function is parameterized, and the location of BUP images.  All good changes, but breaking changes none-the-less.  Chalk that up to maybe not the best first pass implementation.  Just deploying the BUP images to DEPLOY_DIR won't solve anything because the overridden class function won't get invoked.  It could be decided that maybe not many have even adopted an external signing server yet for generating & signing the capsules (it just went last year) in which case the PR could be merged.  Not sure the best way to try to communicate the change other than announcing on the gitter channel so people don't get caught by surprise.